ledger/error/src/main/scala/com/daml/error/definitions/TransactionError.scala

```diff
@@ -2,115 +2,55 @@
 // SPDX-License-Identifier: Apache-2.0
 
 package com.daml.error.definitions
-import com.daml.error.ErrorCode.{formatContextAsString, truncateResourceForTransport}
-import com.daml.error.{BaseError, ErrorCode, ContextualizedErrorLogger}
-import com.daml.ledger.participant.state.v2.Update.CommandRejected.{
-  FinalReason,
-  RejectionReasonTemplate,
-}
-import com.google.rpc.status.{Status => RpcStatus}
-import io.grpc.{Status, StatusRuntimeException}
-
-trait TransactionError extends BaseError {
-  def createRejection(
-      correlationId: Option[String]
-  )(implicit loggingContext: ContextualizedErrorLogger): RejectionReasonTemplate = {
-    FinalReason(rpcStatus(correlationId))
-  }
-
-  // Determines the value of the `definite_answer` key in the error details
-  def definiteAnswer: Boolean = false
-
-  final override def definiteAnswerO: Option[Boolean] = Some(definiteAnswer)
 
-  def rpcStatus(
-      correlationId: Option[String]
-  )(implicit loggingContext: ContextualizedErrorLogger): RpcStatus =
-    _rpcStatus(None, correlationId)
-
-  // TODO error codes: this impl vs. GrpcStatus.toProto
-  def _rpcStatus(
```

Comment:
This one is getting a complete new implementation here https://github.com/digital-asset/daml/pull/13185/files?diff=unified&w=1#r823518674